### ISSUE TITLE
Inconsistent slicing of base values and values of the Explanation object sliced to visualize feature contributions towards a class in a multiclass scenario leading to error in visualizing text plots 

### ISSUE THREAD

**Issue by ryserrao** [shap/shap#1459]
import os
import shutil
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import transformers
from transformers import AutoTokenizer, AutoModelForSequenceClassification
import shap
import scipy as sp
import nlp
import torch

### Load model, tokenizer and data
tokenizer = AutoTokenizer.from_pretrained("lordtt13/emo-mobilebert",use_fast=True)
model = AutoModelForSequenceClassification.from_pretrained("lordtt13/emo-mobilebert").cuda()
train, test = nlp.load_dataset("emo", split = ["train", "test"])

### define a prediction function
def f(x):
    tv = torch.tensor([tokenizer.encode(v, pad_to_max_length=True, max_length=128,truncation=True) for v in x]).cuda()
    attention_mask = (tv!=0).type(torch.int64).cuda()
    outputs = model(tv,attention_mask=attention_mask)[0].detach().cpu().numpy()
    scores = (np.exp(outputs).T / np.exp(outputs).sum(-1)).T
    val = sp.special.logit(scores) 
    return val

### build an explainer using a token masker
explainer = shap.Explainer(f, tokenizer,output_names=['Other','Happy','Sad','Angry'])
shap_values = explainer(train['text'][:50])

### visualizing feature attributions in text plots for the first 3 sentences with respect to class 1
shap.plots.text(shap_values[0:3][:,1])

Error:
![err_shap_text](https://user-images.githubusercontent.com/17132636/93905225-b810f800-fcc8-11ea-9e70-d1ef90acf157.JPG)

@slundberg
